photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -14,28 +14,212 @@
+    // :+ 1.0 is because we need to add that column to make up a correct design matrix, including that 1.0
+    // to multiply by the intercept term.
+    val label = GameConverters.FieldNames.RESPONSE
+    val trainingData: DataFrame = new SQLContext(sc)
+      .createDataFrame(labeledPoints
+        .map { point: LabeledPoint =>
+          val x = Vectors.dense(point.features.toDenseVector.toArray :+ 1.0).toSparse
+          (point.label, x) })
+      .toDF(label, featureShardId)
+
+    val stats = BasicStatisticalSummary(trainingData.select(featureShardId).map(_.getAs[SparkVector](0)))
+
+    // We set args for the GAMEEstimator - only: so this is the minimum set of params required by GAMEEstimator
+    // Default number of passes over the coordinates (numIterations) is 1, which is all we need if
+    // we have a single fixed effect model
+    val args = Map[String, String](
+      "task-type" -> TaskType.LINEAR_REGRESSION.toString,
+      "feature-shard-id-to-feature-section-keys-map" -> s"$featureShardId:${featureNames.mkString(",")}",
+      "fixed-effect-data-configurations" -> s"$coordinateId:$featureShardId,1",
+      "fixed-effect-optimization-configurations" -> s"$coordinateId:100,1e-11,0.3,1,LBFGS,l2",
+      "updating-sequence" -> coordinateId,
+      //"normalization-type" -> NormalizationType.NONE.toString, // not required
+      "train-input-dirs" -> "", // required by GAMEParams parser, but not used in GAMEEstimator
+      "validate-input-dirs" -> "", // required by GAMEParams parser, but not used in GAMEEstimator
+      "output-dir" -> "", // required by GAMEParams parser, but not used in GAMEEstimator
+      "feature-name-and-term-set-path" -> "")  // required by GAMEParams parser, but not used in GAMEEstimator
+    val params = GameParams.parseFromCommandLine(CommonTestUtils.argArray(args))
+
+    // Compute normalization contexts based on statistics of the training data for this (unique) feature shard
+    val normalizationContexts: Option[Map[String, NormalizationContext]] =
+      Some(Map((featureShardId, stats))
+        .mapValues { featureShardStats =>
+          val intercept: Option[Int] = featureIndexMap.get(GLMSuite.INTERCEPT_NAME_TERM)
+          NormalizationContext(params.normalizationType, featureShardStats, intercept)
+        })
+
+    // Create GAMEEstimator and fit model
+    // Returns (model, evaluation, optimizer config)
+    val models: Seq[(GAMEModel, Option[EvaluationResults], String)] =
+        createEstimator(params, "simpleTest")._1.fit(trainingData, validationData = None, normalizationContexts)
+
+    val model = models.head._1.getModel(coordinateId).head.asInstanceOf[FixedEffectModel].model
+
+    // Reference values from scikit-learn
+    assertEquals(model.coefficients.means(0), 0.3215554473500486, 1e-12)
+    assertEquals(model.coefficients.means(1), 0.17904355431985355, 1e-12)
+    assertEquals(model.coefficients.means(2), 0.4122241763914806, 1e-12)
+  }
```

Comment:
This tests no normalization - will there be similar tests for the various kinds of normalization?